winml/lib/Api.Image/TensorToVideoFrameConverter.cpp

```diff
@@ -30,15 +32,45 @@ class GPUTensorToDX12TextureTelemetryEvent {
         TraceLoggingOpcode(EVENT_TRACE_TYPE_START),
         TraceLoggingHexInt32(tensorDesc.channelType, "Type"),
         TraceLoggingInt64(tensorDesc.sizes[2], "Height"),
-        TraceLoggingInt64(tensorDesc.sizes[3], "Width"));
+        TraceLoggingInt64(tensorDesc.sizes[3], "Width"),
+        TelemetryPrivacyDataTag(PDT_ProductAndServiceUsage),
+        TraceLoggingKeyword(MICROSOFT_KEYWORD_MEASURES));
```

Comment:
Do we want to view the data in a app session granularity? If so, we need to add AppSessionGuid to telemetry data so it can be joined with other data such as device/gpu/winml info.